examples/web-getting-deployed/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
```

Comment:
a general question about this file. I'm not familiar with it.

Is it generated by some tool/task?

Reply:
This is for deploying in a WAR file, an alternative way to run instead of running from `main()`

Reply:
This file, `web.xml`,  is part of the [WAR standard](https://en.wikipedia.org/wiki/WAR_(file_format)) (mentioned under _Content and Structure_).  In this case the content was adapted from the official [Ktor WAR packaging example](https://ktor.io/docs/war.html).